### PR TITLE
Fixed issue #18183: Date filter at statistics is not applied correctly

### PR DESCRIPTION
**Pull request by gabrieljenik**
Making the equal filter to use a date picker (just date).
Later on the PHP side, we query by columnanme in between the start and midnight of the filter date.

### CODE REVIEW

File: application/views/admin/export/statistics_subviews/_question.php [hunk 1]
```diff
@@ -277,7 +277,7 @@
                     'id' => $myfield3,
                     'value' => isset($_POST[$myfield3]) ? $_POST[$myfield3] : '',
                     'pluginOptions' => array(
-                        'format' => $dateformatdetails['jsdate'] . " HH:mm",
```
Comment (Shnoulle) [inline application/views/admin/export/statistics_subviews/_question.php:280]: Keep it ?

**Comment by Shnoulle** [comment]
Fixed for eq but font for less and more ? https://github.com/LimeSurvey/LimeSurvey/blob/415fd87e020cfb156c736bb4dc06c9e22fac25c1/application/helpers/admin/statistics_helper.php#L473-L480

@maziminke issue was : « E.g. if I define to show data sets with a date between January 1st, 2021 and December 31st, 2021»

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
@maziminke time for testing ?

**Comment by maziminke** [comment]
Looks good to me!

**Pull request merged**